backend/auth/Auth.ts

```diff
@@ -21,33 +39,36 @@ router.use(
 );
 
 router.use(passport.initialize());
+router.use(passport.session());
 
 router.post(
   "/login",
   (req, res, next) => {
-    console.log("POST LOGIN");
     next();
   },
-  passport.authenticate("custom", { successRedirect: "/create-noun" })
+  passport.authenticate("local", { successRedirect: "/create-noun" }) // change or remove redirect later
 );
 
 router.get("/login", renderWebApp);
 
 router.use("/", async (req, res, next) => {
   if (req.session && req.session.passport && req.session.passport.user.id) {
     return next();
+  } else if (process.env.IS_RUNNING_LOCALLY) {
+    const localUser = await findOrCreateLocalUser(LOCAL_DEV_USER_EMAIL);
+    loginPromise(localUser, req)
```

Comment:
Let's use await here, since we're already in an async function. Also, it might be best not to send the actual error, including stacktrace, to the browser in the response due to security risks. Showing a random person a stacktrace could give them hints about how to hack the system. In general, I usually only send the full stacktrace to the browser in local development, not deployed (see https://github.com/JustUtahCoders/comunidades-unidas-internal/blob/6ced64d696290d2da4875f7a95e927be6d28a144/backend/server.js#L68-L72)

```suggestion
    try {
      await loginPromise(localUser, req)
      next()
    } catch (err) {
      console.error(err);
      res.status(500).send("Server error during login")
    }
```

Reply:
I like using the await.  But I thought we were in a local development context, no?

Reply:
Well right now we're developing the software but this code will be deployed to an environment once we start giving away / selling Flax.

Reply:
Ahh, I see.  That makes sense to me now.  Thank you.